### PR TITLE
feat: add network overlay icon to dapp positions

### PR DESCRIPTION
**Pull request by silasbw**

### CODE REVIEW

File: src/tokens/PositionIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import FastImage from 'react-native-fast-image'
+import { useSelector } from 'src/redux/hooks'
+import { networksIconSelector } from 'src/tokens/selectors'
+import { Spacing } from 'src/styles/styles'
+import { Position } from 'src/positions/types'
+
+const networkImageSize = 12
+const networkImagePosition = 20
+
+export const PositionIcon = ({ position, testID }: { position: Position; testID?: string }) => {
```
Comment (satish-ravi) [inline src/tokens/PositionIcon.tsx:12]: Would be nice to use https://github.com/valora-inc/wallet/blob/main/src/components/IconWithNetworkBadge.tsx for this, although would need parameterizing size, can maybe add a TODO linking to ACT-1137

**Review by satish-ravi — APPROVED** [review]

File: src/tokens/PositionIcon.tsx (hunk 1 above)
Comment (silasbw) [reply src/tokens/PositionIcon.tsx:12]: Doh -- i missed that one.  I'll add the TODO and then take a pass at a follow up PR.

File: src/tokens/PositionIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
```
Comment (jeanregisser) [inline src/tokens/PositionIcon.tsx:12]: nit: easier to parse visually and less chars
```suggestion
export function PositionIcon({ position, testID }: { position: Position; testID?: string }) {
```

**Review by jeanregisser — APPROVED** [review]
🙌

**Pull request merged**